common/src/commonMain/kotlin/org/hisp/dhis/common/App.kt

```diff
@@ -122,6 +123,7 @@ fun Main() {
             Components.BUTTON_BLOCK -> ButtonBlockScreen()
             Components.ICON_CARDS -> IconCardsScreen()
             Components.BOTTOM_SHEET_HEADER -> BottomSheetHeaderScreen()
+            Components.BOTTOM_SHEET -> BottomSheetScreen()
```

Comment:
Could we Merge BottomSheet and header in the single screen. I think this might grow too much, is better to unify components. What do you think?

Reply:
They were merged but Marcos asked me two separate them